trajectory_utils/include/trajectory_utils/conversions/conversions.h

```diff
@@ -40,9 +40,11 @@ void speed_to_time(const std::vector<double>& downtrack, const std::vector<doubl
  * \param initial_speed 
  * \param speeds Output parameter which points to the vector which will store the speeds at each point. 
  *              The first speed will always be initial_speed
+ * \param isStopandWait A boolean vector for all points along downtrack, which defines whether they are part of the stop and wait maneuver or not
+ * \param decel_jerk
  */
 void time_to_speed(const std::vector<double>& downtrack, const std::vector<double>& times, double initial_speed,
-                   std::vector<double>* speeds);
+                   std::vector<double>* speeds , std::vector<bool> isStopandWait, double decel_jerk);
```

Comment:
I feel like this design choice is pretty fragile. It takes a very general method and adds alot of complexity to its use. 
I think a more reusable approach would be to have a time_to_speed_constant_accel method to replace the old time_to_speed and a time_to_speed_constant_jerk method. 
Then the caller simply selects which method to use based on the provided trajectory. If the pure_pursuit_wrapper plugin is duplicated into the pure_pursuit_jerk_plugin this change becomes trivial as one implementation uses one function and the other does not.